### PR TITLE
convert documentation to reStructured Text

### LINKED ISSUE #32: reST/readthedocs documentation

### PR DESCRIPTION
**Pull request by classabbyamp**
can be generated by doing `make html` in the docs directory.

Fixes #32 

### Generated documentation
image

### CODE REVIEW

**Comment by classabbyamp** [comment]
hopefully I got everything into the docs

File: ctyparser/bigcty.py [hunk 1]
```diff
@@ -63,17 +48,20 @@ class BigCty(collections.abc.Mapping):
                                  (?:~(?P<tz>[+-]?\d+(?:\.\d+)?)~)?""", re.X)
 
     def __init__(self, file_path: Union[str, os.PathLike, None] = None):
+        """Can be initialised with data by passing the path to a valid ``cty.json`` file to the constructor.
+        """
```
Comment (0x5c) [inline ctyparser/bigcty.py]:
```suggestion
```
Docstrings are superfluous for `__init__()`.

File: ctyparser/bigcty.py [hunk 2]
```diff
@@ -197,6 +191,9 @@ def __iter__(self):
     # --- Standard methods we should all implement ---
     # str(): Simply return what it would be for the underlaying dict
     def __str__(self):
+        """
+        :return: str(data)
+        """
```
Comment (0x5c) [inline ctyparser/bigcty.py]:
```suggestion
```
Should not have a docstring

**Review by 0x5c — CHANGES_REQUESTED** [review]

File: ctyparser/bigcty.py (hunk 1 above)
Comment (classabbyamp) [reply ctyparser/bigcty.py]: fixed

File: ctyparser/bigcty.py (hunk 2 above)
Comment (classabbyamp) [reply ctyparser/bigcty.py]: fixed

**Review by 0x5c — APPROVED** [review]

**Pull request merged**